tests/virt_autotest/kubevirt_tests_agent.pm

```diff
@@ -106,8 +106,11 @@ sub rke2_agent_setup {
     $self->check_service_status();
     assert_script_run("grep static /var/lib/kubelet/cpu_manager_state");
 
+    # Workaround for failure 'MountVolume.SetUp failed for volume "local-storage" : mkdir /mnt/local-storage: read-only file system'
+    assert_script_run('mkdir -p /root/tmp && mount -o bind /root/tmp /mnt') if (is_transactional);
```

Comment:
Does this mean your new volume dir is /root/tmp ?

Reply:
Yes, some test files will be created under /mnt during test running, and the directory is read-only. So it's a workaround for sle-micro.